### PR TITLE
Ps/just add test celery

### PR DESCRIPTION
**Pull request by Pastromhaug**
- Got celery up and running in the test app

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -28,11 +28,11 @@ services:
       TOX_WORK_DIR: /tox
       PYTHON_VERSION: py37
 
-  test-admin:
+  admin:
     build:
       context: .
       args:
-        IMAGE: 'python:3.7-slim-stretch'
+        IMAGE: 'python:3.6-stretch'
```
Comment (Pastromhaug) [inline docker-compose.yml]: turns out celery doesn't work with python 3.7 yet

Comment (mattbriancon) [reply docker-compose.yml]: Yeah neither does Django 1.11 :(

Comment (mattbriancon) [reply docker-compose.yml]: But this can still be "slim", yeah?

**Review by mattbriancon — APPROVED** [review]
wip
wip
wip
wip
wip
🚢

**Review by tayloraburgess — APPROVED** [review]

**Pull request merged**